src/Authentication/Controllers/SystemUserController.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Altinn.Platform.Authentication.Model;
+using Altinn.Platform.Authentication.Services.Interfaces;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Altinn.Platform.Authentication.Controllers
+{
+    /// <summary>
+    /// CRUD API for the System User 
+    /// </summary>
+    /// [Authorize]
+    [Route("authentication/api/v1/systemuser")]
+    [ApiController]
+    public class SystemUserController : ControllerBase
+    {
+        private readonly ISystemUserService _systemUserService;
+
+        /// <summary>
+        /// Constructor
+        /// </summary>
+        /// <param name="systemUserService">The SystemUserService supports this API specifically.</param>
+        public SystemUserController(ISystemUserService systemUserService)
+        {
+            _systemUserService = systemUserService;
+        }
+
+        /// <summary>
+        /// Returns the list of SystemUsers this PartyID has registered
+        /// </summary>
+        /// <returns></returns>
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status404NotFound)]
+        [HttpGet("list/{partyId}")]
+        public async Task<ActionResult> GetListOfSystemUsersPartyHas(int partyId)
```

Comment:
Here i think partyid can be a parameter. Then the url looks like authentication/api/v1/systemusers?partyid=xxxx